src/winetricks

```diff
@@ -15274,9 +15276,19 @@ load_ie8()
         w_try mv "${W_PROGRAMS_X86_UNIX}/Internet Explorer/iexplore.exe" "${W_PROGRAMS_X86_UNIX}/Internet Explorer/iexplore.exe.bak"
     fi
 
+    if [ "${W_ARCH}" = "win64" ]; then
+        if [ -f "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe" ]; then
+            w_try mv "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe" "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe.bak"
+        fi
+    fi
```

Comment:
I'm not 100% sure what is going on here. Note the test for `"$W_SYSTEM32_DLLS"/plugin.ocx`, and with win64 the test for `"${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe"` which is what qwertychouskie did in his PR.
Maybe a better comment would be `"# Move builtin iexplore.exe to iexplore.exe.bak"`, instead of `"# Unregister Wine IE"`.